core/src/com/unciv/logic/automation/civilization/DiplomacyAutomation.kt

```diff
@@ -197,17 +197,17 @@ object DiplomacyAutomation {
     fun wantsToSignDefensivePact(civInfo: Civilization, otherCiv: Civilization): Boolean {
         val diploManager = civInfo.getDiplomacyManager(otherCiv)
         if (diploManager.hasFlag(DiplomacyFlags.DeclinedDefensivePact)) return false
-        if (diploManager.isRelationshipLevelLT(RelationshipLevel.Ally)) return false
+        if (diploManager.opinionOfOtherCiv() < 60f) return false
```

Comment:
Changed from 80 to 60, they are more likely to accept then they are to send the offer.